lib/asbind-instance/asbind-instance.js

```diff
@@ -30,8 +30,8 @@ async function compileStreaming(source) {
   if (WebAssembly.compileStreaming) {
     return WebAssembly.compileStreaming(source);
   }
-  source = await Promise.resolve(souce);
-  if (source instanceof Response) {
+  source = await Promise.resolve(source);
```

Comment:
Oops, didn't know I had a type here :joy:

Reply:
This was my typo, not yours 😅